benchmark_runner/common/clouds/IBM/ibm_operations.py

```diff
@@ -209,7 +208,7 @@ def version_already_installed(self):
         """
         if self.get_ocp_server_version() == 'null':
             return False
-        elif self.LATEST_VERSION in self.__install_ocp_version and self.get_ocp_server_version() == self.__get_latest_version():
+        elif self.LATEST_VERSION in self.__install_ocp_version and self.get_ocp_server_version() == self.__get_installation_version():
```

Comment:
LATEST_VERSION appears to be the fixed string `latest`.  I'm not quite sure how that's working here?

Reply:
What do you mean, LATEST instead of LATEST_VERSION ?

Reply:
That wasn't what I meant, and no need to change it.  I'm trying to work out what happens if the version specified is `latest`, how that gets translated into a specific version.

Reply:
This is already working. Please let me know if you have more comments.

Reply:
I'll approve it as I don't believe you have made any material change, but I would like to understand how specifying `latest` works.

Reply:
Before I approve it -- I notice that the current diff has a lot of changes to various JSON files.  What's that about?

Reply:
I have update our grafana dashboards, adding it into description.

Reply:
That ideally should be a separate PR, but I'll OK this.  I've updated the title too.